banzai/context.py

```diff
@@ -35,13 +35,13 @@ def __init__(self, command_line_args, settings,
             elif variable != 'command_line_args':
                 super(PipelineContext, self).__setattr__(variable, local_variables[variable])
 
-        for keyword in vars(command_line_args):
-            super(PipelineContext, self).__setattr__(keyword, getattr(command_line_args, keyword))
-
         for key, value in dict(inspect.getmembers(settings)).items():
             if not key.startswith('_'):
```

Comment:
I would add a comment in settings.py to say that anything prefaced with an underscore will get into pipeline context.

Reply:
Done